docs/aks-cheatsheet.md

```diff
@@ -0,0 +1,152 @@
+# Azure Kubernetes Service / AKS cheatsheet
+
+If you don't have access to AKS yet see [Developer on-boarding](/docs/developer-onboarding.md).
+
+All examples below show qa usage and you should adapt accordingly.
+
+## Authentication
+
+### Raising a PIM request
+
+You need to activate the role in the desired cluster below:
+https://portal.azure.com/?Microsoft_Azure_PIMCommon=true#view/Microsoft_Azure_PIMCommon/ActivationMenuBlade/~/azurerbac
+
+Example: Activate `*-teacher-services-cloud-test`. It will be approved automatically after a few seconds
```

Comment:
Is there a way to do this from the command-line? (I remember finding the Azure Web interface very confusing!)

Reply:
No, only if we make a command that called open that url 😅